### DISCUSSION TITLE
Inappropriate completion suggestions

### DISCUSSION THREAD

**Discussion by ovidiuiliescu** [community/community#110977] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

Sometimes Copilot goes off the rails and suggests some rather unsavory completions.

![image](https://github.com/community/community/assets/1020334/1c0211f7-3965-4167-8dff-6257ecaed06d)

![image](https://github.com/community/community/assets/1020334/2229ed39-76ef-4255-81c3-e68cbf679429)

![image](https://github.com/community/community/assets/1020334/39b6ecc9-5e5e-432d-96be-6a4676425683)
![image](https://github.com/community/community/assets/1020334/209dc13f-45a4-4ab4-aad9-0d9227f26aa3)

**Comment by davecheney** [comment, marked as answer]
@ovidiuiliescu i'm very sorry that you experienced this. Please contact support https://support.github.com/contact who will collect some debug information from you and escalate this issue to the appropriate team.
